subsys/shell/shell.c

```diff
@@ -121,17 +134,7 @@ static void shell_write(const struct shell *shell, const void *data,
 		length -= tmp_cnt;
 		if (tmp_cnt == 0 &&
 		    (shell->ctx->state != SHELL_STATE_PANIC_MODE_ACTIVE)) {
-			/* todo  semaphore pend*/
```

Comment:
what happened with `todo`?

Reply:
`todo` was outdated and should have been removed when `k_poll` was added